### PR TITLE
refactor(api-client, app): activate getCsvFiles function

### PR DESCRIPTION
**Pull request by koji**
# Overview
activate getCsvFiles function for ODD Choose Csv File screen.

this pr is picked frontend changes from our demo repo.

Screenshot 2024-07-25 at 2 33 55 PM

use id for csv files on robot and use file path for csv files on usb since as always radio button's id must be unique.

close AUTH-470

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [ ] order items by created data

File: app/src/organisms/ChooseRobotToRunProtocolSlideout/useCreateRunFromProtocol.ts
Comment (koji) [inline app/src/organisms/ChooseRobotToRunProtocolSlideout/useCreateRunFromProtocol.ts:1]: needed to pass runTimeParameterFiles to the robot-server

File: app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx [hunk 1]
```diff
@@ -94,16 +94,17 @@ export function ChooseCsvFile({
...
-                    key={csv.id}
-                    data-testid={csv.id}
-                    buttonLabel={csv.name}
-                    buttonValue={`${csv.id}`}
-                    onChange={() => {
-                      setCsvFileSelected({ id: csv.id, fileName: csv.name })
-                    }}
-                    isSelected={csvFileSelected?.id === csv.id}
-                  />
+                  <React.Fragment key={csv.id}>
```
Comment (jerader) [inline app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx:119]: we need both the key here and the id in `RadioButton`? I would think the key in the fragment would be enough

File: api-client/src/protocols/getCsvFiles.ts [hunk 2]
```diff
@@ -1,43 +1,17 @@
...
+  return request<UploadedCsvFilesResponse>(
     GET,
     `/protocols/${protocolId}/dataFiles`,
     null,
     config
   )
-}
-   */
-
-// ToDo (kk:06/14/2024) remove when activate the above code
```
Comment (jerader) [inline api-client/src/protocols/getCsvFiles.ts:21]: 🗑️ nice!!!

**Review by jerader — APPROVED** [review]
code makes sense to me, just left a small question

File: app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx:119]: only one is enough and that is the reason why I moved the key from `RadioButton` to `React.Fragment`.

**Pull request merged**